.travis.yml

```diff
@@ -19,8 +19,6 @@ addons:
     - mysql-client-5.6
 
 env:
-  global:
-    - NODE_ENV=test
```

Comment:
This was previously being overridden by `NODE_ENV=dev` specified in the definition of `npm run test-ci`.